tools/swift-format/Tests/SwiftFormatTests/NeverUseForceTryTests.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import XCTest
+import SwiftSyntax
+
+@testable import Rules
+
+public class NeverUseForceTryTests: DiagnosingTestCase {
+  public func testInvalidTryExpression() {
+    let input =
```

Comment:
Can you add another test that imports XCTest and make sure the diagnostic is not thrown?